libs/designer-ui/src/lib/floatingactionmenu/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { KeyCodes } from '@fluentui/react';
+import { useBoolean } from '@fluentui/react-hooks';
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+export enum DynamicallyAddedParameterType {
```

Comment:
Perhaps we'll move this enum into a new file for `DynamicallyAddedParameter` once that component gets added? Put it here for now

Reply:
Please don't use typescript enums. It'll break the babel typescript compiler used to build the production version. Use key/value constants instead.

Reply:
changed to be enum strings instead of number, as this is how other files in designer-ui seem to be doing it

Reply:
I mean don't use the enum keyword at all. It does not exist in JS and it generates code. It should not be used in the codebase. Here is an article on valid alternatives: https://maxheiber.medium.com/alternatives-to-typescript-enums-50e4c16600b1

Reply:
Thank you for providing the pointer to the article. I see now.

Removed enum in latest iteration